### ISSUE TITLE
 more normalization for component loaders

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#4617]
**Is your feature request related to a problem? Please describe.**

Necessary roles and the temp table name are replicated in the UI and API components.

**Describe what you're trying to accomplish** 

Remove opportunity to introduce typos and inconsistencies.

**Describe the solution you'd like**

Add to the component loader handler table, use that everywhere.

**Comment by dustymc** [comment]
Unless there's some reason not to, I'm going to (probably slowly) grant access to component loaders to data_entry, and move the restrictions to updates. That will allow users who entered "extras" to view them, although they'll still need higher-level help to update.

**Comment by Jegelewicz** [comment]
OOOOHHHH! So instead of download/edit/upload, I could fix the misspelled agent name right in the loader?

**Comment by dustymc** [comment]
No, I'm just trying to normalize access (mostly so I can keep up with what's what, but transparency is always nice, see https://github.com/ArctosDB/arctos/issues/4604#issuecomment-1124345022).

Edit is https://github.com/ArctosDB/arctos/issues/4583, and I don't think it changes anything with this - if you can update status then you could download/change/update, so there's no reason you couldn't just update inline instead. (Or maybe there is, it now looks like the user has entered the thing you just changed, but that still looks like a conversation for over there, I think??)
